PYIC-8706: Content for Experian KBV CRI error

AmritSidhu opened the pull request:
## Proposed changes
### What changed

- Implement new content page for Experian KBV CRI and routing user to F2F CRI in case user selects post office as alternative identity route under “What would you like to do” .

### Why did it change

-  This will ensure users can recover and continue their identity journey.

### Issue tracking
<!-- Jira ticket & other docs, like RFCs -->

- [PYIC-8706](https://govukverify.atlassian.net/browse/PYIC-8706)

## Checklists

- [ ] READMEs and documentation up-to-date
- [ ] Browser/ unit/ Selenium tests have been written/ updated
- [ ] No risk of exposure: PII, credentials, etc through logs/ code
- [ ] Ensure added/updated routes have CSRF protection if required


[PYIC-8706]: https://govukverify.atlassian.net/browse/PYIC-8706?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

AmritSidhu committed:
PYIC-8706: Content for Routing for Experian KBV CRI error

new content page for Experian KBV CRI and routing user to F2F CRI in case user selects post office as alternative identity route

AmritSidhu committed:
PYIC-8706: Fix broken context content

Wrap content reliant on context in if statements

DanCorderIPV commented on views/ipv/page/sorry-technical-problem.njk:
```diff
@@ -37,7 +37,12 @@
         id: 'whatsNeededToUseTheApp'
     }) }}
 
-    <p class="govuk-body">{{ 'pages.sorryTechnicalProblem.content.p4' | translateWithContext(context) }}</p>
```
I think it would be neater to add a `p4KbvCriError` to the translation file and leave this as it was

AmritSidhu committed:
PYIC-8706: Simplify the html for sorry-technical-problem page

By having a default for the kbv dyanmic context it avoids having messy if statements
